DevCycle.SDK.Server.Local/Api/DVCLocalClient.cs

```diff
@@ -122,8 +122,8 @@ public Variable<T> Variable<T>(User user, string key, T defaultValue)
             {
                 try
                 {
-                    ReadOnlyVariable<object> readOnlyVariable = config.Variables.Get(key);
-                    existingVariable = new Variable<T>(readOnlyVariable, defaultValue);
+                    ReadOnlyVariable<object> readOnlyVariable = config.Variables[key];
```

Comment:
With direct dict access; does this still handle safely with a missing kv pair for the `key`?

Reply:
Though I guess this is now the same because we didn't use `GetOrDefault`

Reply:
it's inside a `ContainsKey` check

Reply:
context ain't the best on github.